src/components/SlateEditor/plugins/file/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { Descendant, Editor, Element } from 'slate';
+import { RenderElementProps } from 'slate-react';
+import Filelist from './Filelist';
+import { createEmbedTag, parseEmbedTag } from '../../../../util/embedTagHelpers';
+import { SlateSerializer } from '../../interfaces';
+import { File } from '../../../../interfaces';
+import { getFileBlock } from './utils';
+
+export const TYPE_FILE = 'file';
+
+export interface FileElement {
+  type: 'file';
+  data: File[];
+  children: Descendant[];
+}
+
+export const fileSerializer: SlateSerializer = {
+  deserialize(el: HTMLElement) {
+    if (el.tagName.toLowerCase() !== 'div') return;
+    if (el.dataset.type !== TYPE_FILE) return;
+    return getFileBlock(
+      el.innerHTML.split(',').map(embed => (parseEmbedTag(embed) as unknown) as File),
+    );
+  },
+  serialize(node: Descendant) {
+    if (!Element.isElement(node)) return;
+    if (node.type !== TYPE_FILE) return;
+    return `<div data-type="file">${node.data.map(file => createEmbedTag(file))}</div>`;
+  },
+};
+
+export const filePlugin = (editor: Editor) => {
```

Comment:
Når blokker plasseres rett etter hverandre skal det plasseres tomme paragrafer mellom. Gjøres med editor.normalizeNode.

Sjekk detailsPlugin/index.tsx, linje 206-238. Der normaliseres det så tom paragraph blir lagt til foran og etter om nødvendig.

Reply:
Gjøres ikke dette i section plugin? Det fungerer i alle fall sånn som det er nå

Reply:
Section plugin ser kun på første og siste blokk i editoren. Om jeg legger ut to fil-lister etter hverandre nå kommer det ikke en tom paragraf mellom de.